execution/executor/src/db_bootstrapper.rs

```diff
@@ -28,13 +26,6 @@ pub fn bootstrap_db_if_empty<V: VMExecutor>(
         .expect("failed to load genesis txn!")
         .clone();
 
-    bootstrap_db_if_empty_impl::<V>(db, genesis)
-}
-
-fn bootstrap_db_if_empty_impl<V: VMExecutor>(
-    db: &DbReaderWriter,
-    genesis: Transaction,
```

Comment:
seems mostly reasonable except that we shouldn't expect a node config, in fact, if you think about it, we only pass in the node config to remove genesis, maybe we could push the txn API into where it is called so we don't pass the config everywhere.

Reply:
yeah taking the config is effectively just to save 5 lines (extracting the genesis txn from the config) on the call site. Adding a `.expect_genesis_txn() -> Result<Transaction>` helper to the Config object will remove the need, not sure if you are happy with that.

Reply:
I can poke at this later, so as to not block you.